pkg/utils/flags.go

```diff
@@ -76,8 +76,7 @@ func (a *StringSetVal) String() string {
 	if *a == nil {
 		return ""
 	}
-
-	vals := make([]string, len(*a), 0)
+	vals := make([]string, 0, len(*a))
```

Comment:
The old implementation caused an error for me. In fact, it makes sense because, in the `make` documentation, I found this `A second integer argument may be provided to specify a different capacity; it must be no smaller than the length`.

Reply:
Oh, apologies, I looked at this too hastily and didn't realize the order of args was swapped.

This is a legit change 👍 How come the old implementation even worked 🧐